data/systemd/snapd.autoimport.service.in

```diff
@@ -2,7 +2,8 @@
 Description=Auto import assertions from block devices
 After=snapd.service snapd.socket snapd.seeded.service
 # don't run on classic
-ConditionKernelCommandLine=snap_core
+ConditionKernelCommandLine=|snap_core
+ConditionKernelCommandLine=|snapd_recovery_mode
```

Comment:
Mmh, should we run some of these only in run mode or not?  it's an orthogonal question but we will have to think it trough, in particular  atm assertions that would be auto-imported at install time will not be copied into the running system.  That's something to think about...

Reply:
Good point, I added a "FIXME:UC20:" to make sure we look into this.